OpenRA.Mods.Common/Traits/Power/Player/PowerManager.cs

```diff
@@ -98,6 +101,9 @@ public void UpdateActor(Actor a)
 
 		public void RemoveActor(Actor a)
 		{
+			if (a.IsInWorld)
```

Comment:
_Don't_ remove this actor from the player's power if it _is_ in the world?
Would you explain for me why we want this behavior, please?

Reply:
This method doesn't remove actors from the world, which means the actor should already be removed from the world when this runs since removing power of an actor that still is in the world is wrong. (The same as adding power of an actor that is not in the world.) I can add a comment to clarify that.